File: C++/Iniciante/1012.cpp
#include <iostream>
#include<iomanip> 
using namespace std;
 
int main(){
    double A, B, C, pi, triangulo, circulo, trapezio, quadrado, retangulo;
	cin >> fixed >> setprecision(2);	cin >> A >> B >> C;
	
	pi = 3.14159;
	triangulo = A*C/2;
	quadrado = B*B;
	trapezio = (A+B)*C/2;
	circulo = pi*C*C;
	retangulo = A*B;
	
    cout << fixed << setprecision(3); 
    cout << "TRIANGULO: " << triangulo << endl;
    cout << "CIRCULO: " << circulo << endl;
    cout << "TRAPEZIO: " << trapezio << endl;
    cout << "QUADRADO: " << quadrado << endl;
    cout << "RETANGULO: " << retangulo << endl;
    return 0;
}//fim do algoritmo

File: Python3/Iniciante/1012.py
# -*- coding: utf-8 -*-

ENTRADA = input()

A = float(ENTRADA.split()[0])
B = float(ENTRADA.split()[1])
C = float(ENTRADA.split()[2])

TRIANGULO_RETANGULO = (A*C)/2
CIRCULO = 3.14159*(C**2)
TRAPEZIO = ((A+B)*C)/2
QUADRADO = B**2
RETANGULO = A*B

print("TRIANGULO: %.3f" % TRIANGULO_RETANGULO)
print("CIRCULO: %.3f" % CIRCULO)
print("TRAPEZIO: %.3f" % TRAPEZIO)
print("QUADRADO: %.3f" % QUADRADO)
print("RETANGULO: %.3f" % RETANGULO)
